sdk/webpubsub/azure-messaging-webpubsubservice/setup.py

```diff
@@ -64,7 +64,7 @@
     install_requires=[
         "isodate<1.0.0,>=0.6.1",
         "azure-core<2.0.0,>=1.24.0",
-        "pyjwt>=1.7.1",
+        "pyjwt>=2.0.0"
```

Comment:
Please update setup.py with reference of https://github.com/Azure/azure-sdk-for-python/blob/7181b93ce68d17189dbe4f054449d25ace3f523c/tools/azure-sdk-tools/packaging_tools/templates/packaging_files/setup.py#L53-L61

Reply:
removed some classifier